dman.d

```diff
@@ -142,7 +142,7 @@ string CHeader(string topic)
     static string[] dmccmds =
     [
         "assert.h",     "complex.h",   "ctype.h",      "fenv.h",
-        "float.h",      "locale.h",    "math.h",       "setjmp.h,"
+        "float.h",      "locale.h",    "math.h",       "setjmp.h,",
```

Comment:
Good catch but I think the comma should also be removed from the string literal.